flink-table/flink-table-planner-blink/src/test/java/org/apache/flink/table/planner/factories/TestValuesTableFactory.java

```diff
@@ -565,14 +562,12 @@ private boolean binaryFilterApplies(CallExpression binExpr, Row row) {
 			}
 		}
 
-		private boolean isComparable(Class<?> clazz) {
-			// validate that literal is comparable
+		private void validateTypeComparable(Class<?> clazz) {
```

Comment:
I think this method should be 
```
private boolean isComparable(Class<?> clazz) {
    return Comparable.class.isAssignableFrom(clazz);
}
```
we do not need throw exception if a class is non-comparable.`TestValuesTableFactory` just does not support non-comparable type. just like the case that `TestValuesTableFactory` just supports `UPPER` and `LOWER` UDF.

Reply:
@godfreyhe , But need it log some warning info here?

Reply:
I don't think so. We can add some java doc at the class level to explain which patterns this class supports

Reply:
good idea +1